SQRLUtilsLib/SQRLCPSServer.cs

```diff
@@ -57,13 +58,44 @@ public class CPSServer
         /// </summary>
         public bool Running = false;
 
+        /// <summary>
+        /// Holds the HTML Content of the CPS Abort page pulled from an internal Resource File
+        /// </summary>
+        private string AbortHTML = "";
```

Comment:
If possible, I would like to keep our coding style consistent throughout the project.
In the client, I have meanwhile harmonized almost all the code to follow a single style, maybe we can do so in the library too:

Private fields should start with underscore and a small letter: `private string _abortHTML = ""; `